packages/datadog-instrumentations/src/mongodb-core.js

```diff
@@ -25,13 +25,19 @@ addHook({ name: 'mongodb-core', versions: ['2 - 3.1.9'] }, Server => {
   return Server
 })
 
-addHook({ name: 'mongodb', versions: ['>=4'], file: 'lib/cmap/connection.js' }, Connection => {
+addHook({ name: 'mongodb', versions: ['>=4 <4.6.0'], file: 'lib/cmap/connection.js' }, Connection => {
   const proto = Connection.Connection.prototype
   shimmer.wrap(proto, 'command', command => wrapConnectionCommand(command, 'command'))
   shimmer.wrap(proto, 'query', query => wrapConnectionCommand(query, 'query'))
   return Connection
 })
 
+addHook({ name: 'mongodb', versions: ['>=4.6.0'], file: 'lib/cmap/connection.js' }, Connection => {
+  const proto = Connection.Connection.prototype
+  shimmer.wrap(proto, 'command', command => wrapConnectionCommand(command, 'command'))
+  return Connection
+})
```

Comment:
Not sure how you prefer to do this, when there's changes in a file such that you need to instrument it in different ways between versions

Reply:
That's actually the way we usually do it.

Do you know why no test is failing even without this change? There should probably be something to ensure that there is no regression in the future and to prove that this change does fix an issue.

Reply:
I just realized that my question didn't quite make sense since the methods were removed so it wouldn't fail anything.